controllers/controllers/services/osbapi/client_test.go

```diff
@@ -619,6 +620,59 @@ var _ = Describe("OSBAPI Client", func() {
 			})
 		})
 
+		Describe("GetServiceBinding", func() {
+			var (
+				bindingResp osbapi.BindingResponse
+				getBindErr  error
+			)
+			BeforeEach(func() {
+				brokerServer.WithResponse(
+					"/v2/service_instances/{instance_id}/service_bindings/{binding_id}",
+					map[string]any{
+						"parameters": map[string]string{
+							"billing-account": "abcde12345",
+						},
+					},
+					http.StatusOK,
+				)
+			})
+			JustBeforeEach(func() {
+				bindingResp, getBindErr = brokerClient.GetServiceBinding(ctx, osbapi.BindPayload{
+					InstanceID: "my-service-instance",
+					BindingID:  "my-binding-id",
+				})
+			})
+
+			It("gets the service binding", func() {
+				Expect(getBindErr).NotTo(HaveOccurred())
+
+				requests := brokerServer.ServedRequests()
+				Expect(requests).To(HaveLen(1))
+				Expect(requests[0].Method).To(Equal(http.MethodGet))
+				Expect(requests[0].URL.Path).To(Equal("/v2/service_instances/my-service-instance/service_bindings/my-binding-id"))
```

Comment:
This test should also check the query parameters (offering and plan id) of the request url